presto-docs/src/main/sphinx/release/release-0.213.rst

```diff
@@ -0,0 +1,99 @@
+=============
+Release 0.213
+=============
+
+General Changes
+---------------
+
+* Fix split scheduling backpressure when plan contains colocated join. Previously, splits
+  for the second and subsequent scan nodes (in scheduling order) were scheduled continuously
+  until completion, rather than pausing due to sufficient pending splits.
+* Fix query execution failure or indefinite hang during grouped execution when all splits
+  for any lifespan are completely filtered out.
+* Fix grouped execution to respect the configured concurrent lifespans per task.
+  Previously, it always used a single lifespan per task.
+* Fix :func:`ST_Distance` function to return ``NULL`` if any of the inputs is an
+  empty geometry as required by the SQL/MM specification.
+* Fix a scenario where too many rows are returned to clients in a single response.
+* Enhance ``SHOW STATS`` to support ``IN`` and ``BETWEEN`` predicates in the
+  ``WHERE`` condition of the ``SELECT`` clause.
+* Do not allow setting invalid property values with :doc:`/sql/set-session`.
+* Disable stats calculator by default as it can cause a planning failure for
+  certain complex queries. It can be enabled with the ``experimental.enable-stats-calculator``
+  configuration property or the ``enable_stats_calculator`` session property.
+* Avoid making wild guesses when estimating filters for joins. Previously, if nothing
+  was known about the filter, a ``0.9`` coefficient was applied as a filter factor.
+  Now, if nothing is known about a filter, the estimate will be unknown. A ``0.9``
+  coefficient will be applied for all additional conjuncts if at least a single
+  conjunct can be reasonably estimated.
+* Improve inference of predicates for inner joins.
+* Improve ``EXPLAIN ANALYZE`` output by including *real* CPU time and scheduled time.
+* Include stats and cost estimates in textual plans created on query completion.
+* Remove transaction from explain plan for indexes.
+* Add ``max_drivers_per_task`` session property, allowing users to limit concurrency by
+  specifying a number lower than the system configured maximum. This can cause the
+  query to run slower and consume less resources.
+* Add ``join-max-broadcast-table-size`` configuration property and
+  ``join_max_broadcast_table_size`` session property to control the maximum estimated size
+  of a table that can be broadcast when using ``AUTOMATIC`` join distribution type (:issue:`11667`).
+* Add :func:`ST_MultiPoint` function to construct multi-point geometry from an array of points.
+* Add :func:`geometry_union` function to efficiently union arrays of geometries.
```

Comment:
Maybe place geometry-related functions together? (`ST_Distance` was several bullets earlier)

Reply:
We try to follow this order for each section: fixes, improvements, additions

Though I'm wondering if we should have a separate section for *Geospatial Changes* like we sometimes do for *Security Changes* when there are several items

Reply:
I'll split these out into a separate section (that still follows the above ordering)